catalog/camel-catalog/src/generated/resources/org/apache/camel/catalog/models/langChain4j.json

```diff
@@ -0,0 +1,23 @@
+{
+  "model": {
+    "kind": "model",
+    "name": "langChain4j",
+    "title": "LangChain4J Tokenizer",
+    "description": "Represents a tokenizer that uses LangChain4j for tokenization",
+    "deprecated": false,
+    "firstVersion": "4.8.0",
+    "label": "tokenizer,transformation,langchain4j",
```

Comment:
Can we use `ai` instead of `langchain4j` so the label matches better that its for ai stuff. It has langchain in the name.

Reply:
Absolutely. I have adjusted it.